
package kz.nitec.shep.service.async;

import javax.xml.ws.WebFault;


/**
 * Ошибка
 *
 * This class was generated by Apache CXF 3.3.0
 * 2021-06-08T11:25:55.832+06:00
 * Generated source version: 3.3.0
 */

@WebFault(name = "sendMessageFault1_sendMessageFault", targetNamespace = "http://bip.bee.kz/AsyncChannel/v10/Types")
public class SendMessageSendMessageFaultMsg extends Exception {

    private kz.nitec.shep.service.async.ErrorInfo sendMessageFault1SendMessageFault;

    public SendMessageSendMessageFaultMsg() {
        super();
    }

    public SendMessageSendMessageFaultMsg(String message) {
        super(message);
    }

    public SendMessageSendMessageFaultMsg(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public SendMessageSendMessageFaultMsg(String message, kz.nitec.shep.service.async.ErrorInfo sendMessageFault1SendMessageFault) {
        super(message);
        this.sendMessageFault1SendMessageFault = sendMessageFault1SendMessageFault;
    }

    public SendMessageSendMessageFaultMsg(String message, kz.nitec.shep.service.async.ErrorInfo sendMessageFault1SendMessageFault, java.lang.Throwable cause) {
        super(message, cause);
        this.sendMessageFault1SendMessageFault = sendMessageFault1SendMessageFault;
    }

    public kz.nitec.shep.service.async.ErrorInfo getFaultInfo() {
        return this.sendMessageFault1SendMessageFault;
    }
}
